### ISSUE TITLE
Version 0.5.0 checklist

### ISSUE THREAD

**Issue by MaxHalford** [creme-ml/creme#178]
Here's a list of main issues I would like to see solved before we make the release:

- [ ] [Word embeddings](https://github.com/creme-ml/creme/issues/148); I think we can mark this as solved once we've implemented at least one word embedding algorithm. GloVe seems like a good candidate.
- [ ] [Batch windows for semi-incremental learning](https://github.com/creme-ml/creme/issues/134); I really like the comment by @johny-c and it would be great if we could implement an API for repeating observations when reading from a stream.
- [ ] [FMRegressor](https://github.com/creme-ml/creme/issues/61)
- [ ] [Focal loss](https://github.com/creme-ml/creme/issues/2)
- [ ] [Bayesian linear regression](https://github.com/creme-ml/creme/issues/65); I have to take care of this. Basically I only have to figure out how to produce prediction intervals and we're good to go.
- [ ] [Gradient boosting](https://github.com/creme-ml/creme/issues/13); @raphaelsty has already implemented AdaBoost, but it would be nice to check if other implementations work better.
- [ ] [Poisson regression](https://github.com/creme-ml/creme/issues/124); I've tried implementing the Poisson loss, but I didn't find a dataset on which it worked better than squared loss. Note that this is [implemented](https://www.tensorflow.org/api_docs/python/tf/keras/losses/poisson) in `tf.keras.losses`.
- [ ] [Exponential smoothing](https://github.com/creme-ml/creme/issues/168)
- [ ] [Birch clustering](https://github.com/creme-ml/creme/issues/81); a fair amount of people have expressed interest in having more online clustering solutions (k-means just isn't good enough)
- [ ] [Online SVM](https://github.com/creme-ml/creme/issues/77); ALMA seems to be a good first method to implement. The main thing is that I'm not sure about the API. Should this be an optimizer or a separate class?. It's fine if we don't solve this issue for the release, but I would like to do some brainstorming on it regarding the API.

There also some issues related to docs and tests which need to be solved:

- [ ] [Improve CHANGELOG](https://github.com/creme-ml/creme/issues/166) 
- [ ] [Improve referencing](https://github.com/creme-ml/creme/issues/158)
- [ ] [Fix codecov for Cython files](https://github.com/creme-ml/creme/issues/167)
- [ ] [Use real datasets in check_estimator](https://github.com/creme-ml/creme/issues/177)

Naturally this doesn't mean other issues can't be worked on too!

**Comment by MaxHalford** [comment]
Closing this because using milestones seems to be the right way to group issues.
